mock-identity-certify-plugin/pom.xml

```diff
@@ -335,6 +335,26 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-antrun-plugin</artifactId>
+                <version>3.1.0</version>
+                <executions>
+                    <execution>
+                        <id>make-jar-executable</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <configuration>
+                            <target>
+                                <!-- Create a shell script to run the JAR file with executable permission -->
+                                <chmod file="${project.build.directory}/*.jar" perm="ugo+rx"/>
+                            </target>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
```

Comment:
one more last minute change, sorry about this.

We have rename the folder and the artifact id to below

from mock-identity-certify-plugin to mock-certify-plugin since this plugin is not only for identity.
I believe this will have changes in the artifactory also.

Reply:
changed